SPECS/grpc/grpc.spec

```diff
@@ -0,0 +1,83 @@
+Summary:        Open source remote procedure call (RPC) framework
+Name:           grpc
+Version:        1.35.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/System
+URL:            https://www.grpc.io
+#Source0:        https://github.com/grpc/grpc/archive/v%{version}/%{name}-%{version}.tar.gz
```

Comment:
Thoughts on merging the two Source0 lines? With the full URL, the tools will still search for `%{name}-%{version}.tar.gz`.

Reply:
Yeah, merged the two